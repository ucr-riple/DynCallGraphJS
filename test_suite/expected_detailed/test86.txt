{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:10:24:14:3)': [
        'Proxy (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:10:24:14:3)'
    ],
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:15:1:15:23)': [
        'anon (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:15:1:15:23)'
    ],
    'anon (Native)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:11:10:13:4)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:12:12:12:24)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:6:19:8:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:12:12:12:38)': [
        'toUpperCase (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test86.js:12:12:12:38)'
    ]
}