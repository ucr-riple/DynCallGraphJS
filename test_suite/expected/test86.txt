{ '(test86.js:10:24:14:3)': [ 'Proxy (Native)' ],
  '(test86.js:15:1:15:23)': [ 'anon (Native)' ],
  'anon (Native)':
   [ '(test86.js:11:10:13:4)' ],
  '(test86.js:12:12:12:24)':
   [ '(test86.js:6:19:8:2)' ],
  '(test86.js:12:12:12:38)': [ 'toUpperCase (Native)' ] }